modules/ocf_tv/manifests/init.pp

```diff
@@ -63,4 +63,11 @@
       User['ocftv'],
     ],
   }
+
+  $pulse_sink = lookup('pulse_sink')
+
+  exec { "pactl set-default-sink ${pulse_sink}":
```

Comment:
we can do the same thing as in ocf_desktop as use augeas, but that's a bit overkill for this I think. We can make it run less frequently as like a daily cronjob or something but this is a very lightweight and idempotent operation.